packages/accounts-app/src/AddAccount/ImportWithPhrase.tsx

```diff
@@ -53,14 +63,29 @@ export function ImportWithPhrase(props: Props): React.ReactElement {
   const [name, setName] = useState('');
   const [password, setPassword] = useState('');
   const [recoveryPhrase, setRecoveryPhrase] = useState('');
+  const [tags, setTags] = useState<Tags>([]);
+  const [tagOptions, setTagOptions] = useState<TagOptions>([
+    { key: '0', text: 'stash', value: 'Stash' },
+    { key: '1', text: 'controller', value: 'Controller' },
+  ]);
+
+  const handleAddTag = (_event: React.SyntheticEvent, { value }: any): void => {
+    setTagOptions([...tagOptions, { key: value, text: value, value }]);
+  };
+
+  const handleOnChange = (_event: React.SyntheticEvent, { value }: any): void => {
+    setTags(value);
+  };
 
   const handleUnlockWithPhrase = (): void => {
     validate({ name, password, recoveryPhrase })
       .chain(({ recoveryPhrase }) =>
         tryCatch2v(
           () => {
             // This is inside tryCatch, because it might fail
-            keyring.createFromUri(recoveryPhrase.trim(), {}, 'sr25519');
+            // addUri(suri: string, password?: string, meta?: KeyringPair$Meta, type?: KeypairType): CreateResult;
```

Comment:
Not sure putting signature as comment is really useful, if I need the signature I can just hover over addUri below

Reply:
my bad, forgot to delete it.